### ISSUE TITLE
RPCQ does not pass along a "**kwargs" dictionary key

### ISSUE THREAD

**Issue by notmgsk** [rigetti/rpcq#78]
On the lisp-side, RPCQ's server checks the incoming `RPCRequest`'s `params` dictionary for a key "**kwargs", and those are passed on to the function specified in the request as keyword args. Ostensibly this is so that in python-world you can form a request with `rpc_request()` and provide arbitrary data that is passed along with the request. The python-side however does not pack those keyword args into the "**kwargs" key.

The definition of `rpc_request()` is
```
def rpc_request(method_name: str, *args, **kwargs) -> rpcq.messages.RPCRequest:
    """
    Create RPC request

    :param method_name: Method name
    :param args: Positional arguments
    :param kwargs: Keyword arguments
    :return: JSON RPC formatted dict
    """
    if args:
        kwargs['*args'] = args

    return rpcq.messages.RPCRequest(
        jsonrpc='2.0',
        id=str(uuid.uuid4()),
        method=method_name,
        params=kwargs
    )
```
and the relevant `server.lisp` is
```
(kwargs-as-plist
 (if (gethash "**kwargs" (|RPCRequest-params| request))
     (loop :for key :being :the :hash-keys :of (gethash "**kwargs" (|RPCRequest-params| request))
             :using (hash-value val)
           :collect (str->lisp-keyword key)
           :collect val)
     nil))
```

**Comment by notmgsk** [comment]
Apologies -- this is actually a pyQuil issue.

**Comment by notmgsk** [comment]
Closed by #80
